cmd/infra/powervs/create.go

```diff
@@ -388,28 +384,43 @@ func (infra *Infra) SetupInfra(ctx context.Context, options *CreateInfraOptions)
 		return fmt.Errorf("cloud connection is not up: %w", err)
 	}
 
+	if err = infra.setupSecrets(options); err != nil {
```

Comment:
Please comment, why it needs to be after the resources created

Reply:
Added a comment, 
please let me know if it is understandable?